cfme/utils/appliance/implementations/ui.py

```diff
@@ -547,7 +547,7 @@ def go(self, _tries=0, *args, **kwargs):
         except Exception as e:
             self.log_message(
                 "Exception raised [{}] whilst checking if already here".format(e), level="error")
-        if not here:
+        if not here or nav_args['force_nav']:
             self.log_message("Prerequisite Needed")
```

Comment:
Let's have more accurate logging here - at least stating 'or force used' in the log message, even better having a separate log message.

Reply:
added